scripts/localstack/setup-s3.sh

```diff
@@ -1,2 +1,4 @@
 #!/usr/bin/env bash
-awslocal s3 mb s3://bucket
+awslocal s3 mb s3://admin
+awslocal s3 mb s3://organization1
+awslocal s3 mb s3://organization2
```

Comment:
Could you explain a bit?
Initially this was creating the default bucket required. Considering the purpose is the same, if the admin is the only user created, his org needs to be created, which is supposed to be the only bucket created. So I don't understand why we'd create 3

Reply:
because once you told me that you didn't want automatic creation of the buckets to avoid cost inflation, so in production this operation would stay manual. In test I put it here